### ISSUE TITLE
per axes logscale options without toplevel logscale option does not show graph

### ISSUE THREAD

**Issue by KennethNielsen** [danvk/dygraphs#867]
Hallo dygraph devs and thanks for an awesome library

I am trying to upgrade to 2.0.0 and I am loving the new per series and per axes options. I have, however, run into a problem. If I understand the docs correctly, I should be able to set logscale in the axes configuration, however, if I do, without setting logscale as a top level option, the graph will disappear.

Consider this fiddle: https://jsfiddle.net/eM2Mg/9987/ It is the standard fiddle for version 2.0.0, where I have edited the data a bit to make it easy to see the difference in log/lin plotting. Now consider the following permutations of: commenting the toplevel logscale=true in or out and boolean value for the per-axes configuration.

Toplevel logscale commented out:
y-logscale true, y2-logscale false -> Y doesn't show, axis correct, Y2 plotted correct
y-logscale false, y2-logscale true -> as above but switched around
y-logscale true, y2-logscale true -> neither graph shows, both axis log
y-logscale false, y2-logscale false -> All good

Toplevel logscale set to true
y-logscale and y2 logscale not set -> Both graphs show as log and correct
y-logscale true, y2-logscale false -> All good
y-logscale false, y2-logscale true -> All good
y-logscale true, y2-logscale true -> All good
y-logscale false, y2-logscale false -> All good

The bottom line seems to be:

1. Without setting logscale to true at the toplevel, setting it per axes will make the graphs on that axis not render
2. Setting logscale to true at the toplevel, will make the graphs be correct if log is specified per axes for all axes (and thus works as a work-around for now).

While I realize that that all the dual y-axis one log and one not is fairly exotic, it is the reason I use dygraphs, because no-one else does it. It should however be noted, that all the observations above also apply to the one y-axis case.

Just for completeness I use Firefox on Ubuntu and the version is 2.0.0.

**Comment by 121watts** [comment]
Same issue here 😢
